api/HTMLFrameSetElement.json

```diff
@@ -142,53 +142,6 @@
           }
         }
       },
-      "onlanguage": {
-        "__compat": {
-          "support": {
-            "chrome": {
-              "version_added": null
-            },
-            "chrome_android": {
-              "version_added": null
-            },
-            "edge": {
-              "version_added": null
-            },
-            "firefox": {
-              "version_added": "32"
```

Comment:
Just to be sure, I tested http://mdn-bcd-collector.appspot.com/tests/api/HTMLFrameSetElement/onlanguage in Firefox 32, and no, it's not there.